cmd/ccboc/main.go

```diff
@@ -249,6 +336,20 @@ func main() {
 						Usage:   "",
 						Action:  getCalculations,
 					},
+
+					{
```

Comment:
lets just have one command that belongs in the calculation command. I would expect for the user to just type `ccboc get calculation results {id}` or `ccboc --logG=4.0 --teff=1000 get calculation results
`

Reply:
@gszasz We could use your insight as a user in this one...